cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -84,6 +84,7 @@ func NewAddCmd(o *AddOrUpdateOptions, flagsFactory cmdcore.FlagsFactory) *cobra.
 		AllowDisableWait: true,
 		DefaultInterval:  1 * time.Second,
 		DefaultTimeout:   5 * time.Minute,
+		WaitByDefault:    o.pkgCmdTreeOpts.WaitByDefault,
```

Comment:
This needs to be added for the update command as well, this is the reason for the failing test.